src/serialize.jl

```diff
@@ -0,0 +1,18 @@
+const pickle = PyNULL()
+
+function get_pickle()
+    if pickle.o == C_NULL
+        copy!(pickle, pyimport("pickle"))
+    end
+
+    return pickle
+end
+
+function Base.serialize(s::AbstractSerializer, pyo::PyObject)
+    Base.serialize_type(s, PyObject)
+    serialize(s, get_pickle()[:dumps](pyo))
+end
+
+function Base.deserialize(s::AbstractSerializer, t::Type{PyObject})
+    get_pickle()[:loads](deserialize(s))
```

Comment:
(This unfortunately makes a copy of the buffer, but I haven't implemented a copy-free Julia pybuffer wrapper around a Julia array yet.)